### PR TITLE
Harvest Temple Spreads & Reds (Normal Mode) - Ai, Fractal Avengers and Vindicators Icons

### PR DESCRIPTION
**Pull request by Linkaaaaa**
- Added decorations for the spread aoes and red void pools to harvest temple normal mode
- Added icons for all Ai npcs except the player-guilt
- Added fractal avenger and vindicators icons

Currently the avengers and vindicators won't show up in the combat replay, should all the trash mobs method in fractals be reworked similar to this screenshot?

Also the icons are on my imgur account now, no longer anonymous

### CODE REVIEW

File: GW2EIEvtcParser/EncounterLogic/Strikes/EndOfDragons/HarvestTemple.cs [hunk 1]
```diff
@@ -20,8 +20,8 @@ public HarvestTemple(int triggerID) : base(triggerID)
             MechanicList.AddRange(new List<Mechanic>
             {
                 // General
-                new PlayerDstEffectMechanic(EffectGUIDs.HarvestTempleSpread, "Spread Bait", new MechanicPlotlySetting(Symbols.Circle, Colors.Yellow), "Spread.B", "Baited spread mechanic", "Spread Bait", 150),
-                new PlayerDstEffectMechanic(EffectGUIDs.HarvestTempleRedPuddleSelect, "Red Bait", new MechanicPlotlySetting(Symbols.Circle, Colors.Red), "Red.B", "Baited red puddle mechanic", "Red Bait", 150),
+                new PlayerDstEffectMechanic(EffectGUIDs.HarvestTempleSpreadNM, "Spread Bait", new MechanicPlotlySetting(Symbols.Circle, Colors.Yellow), "Spread.B", "Baited spread mechanic", "Spread Bait", 150),
```
Comment (EliphasNUIT) [inline GW2EIEvtcParser/EncounterLogic/Strikes/EndOfDragons/HarvestTemple.cs:23]: Those two mechanics should be completed with the other mode's IDs.

**Comment by EliphasNUIT** [comment]
> Currently the avengers and vindicators won't show up in the combat replay, should all the trash mobs method in fractals be reworked similar to this screenshot?
Well, if you want them to appear on fractal encounters overriding that API, you don't have much choice ^^

**Comment by EliphasNUIT** [comment]
> Also the icons are on my imgur account now, no longer anonymous

Are you talking about the new ones in the PR specifically or all images you have added so far?

**Comment by Linkaaaaa** [comment]
> > Also the icons are on my imgur account now, no longer anonymous
> 
> Are you talking about the new ones in the PR specifically or all images you have added so far?

these new ones :D

File: GW2EIEvtcParser/EncounterLogic/Strikes/EndOfDragons/HarvestTemple.cs (hunk 1 above)
Comment (Linkaaaaa) [reply GW2EIEvtcParser/EncounterLogic/Strikes/EndOfDragons/HarvestTemple.cs]: Had missed that, fixed

**Comment by EliphasNUIT** [comment]
Thanks!

**Pull request merged**